src/projects/markdown-previewer/index.jsx

```diff
@@ -14,11 +14,15 @@ marked.setOptions({
   breaks: true
 });
 
-// INSERTS target="_blank" INTO HREF TAGS (required for codepen links)
+// INSERTS target="_blank" INTO HREF TAGS (required for Codepen links)
 const renderer = new marked.Renderer();
-renderer.link = function(href, title, text) {
+renderer.link = function (href, title, text) {
   return `<a target="_blank" href="${href}">${text}` + '</a>';
```

Comment:
If you agree with the change below, would you mind also simplifying this line a bit?:

```suggestion
  return `<a target="_blank" href="${href}">${text}</a>`;
```